app/resources/ampstart.html.template

```diff
@@ -0,0 +1,50 @@
+{# Copyright 2010 Google Inc.  Licensed under the Apache License, Version   #}
+{# 2.0 (the "License"); you may not use this file except in compliance with #}
+{# the License.  You may obtain a copy of the License at:                   #}
+{#     http://www.apache.org/licenses/LICENSE-2.0                           #}
+{# Unless required by applicable law or agreed to in writing, software      #}
+{# distributed under the License is distributed on an "AS IS" BASIS,        #}
+{# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. #}
+{# See the License for the specific language governing permissions and      #}
+{# limitations under the License.                                           #}
+
+{# Template for the repository starting page.                               #}
+
+{% extends "ampapp-base.html.template" %}
+{% load i18n %}
+
+{% block content %}
+
+  <div class="languages">
+    <a href="{{env.start_url}}">Other Languages</a>
+  </div>
+
+  <div class="main">
```

Comment:
The content of this DIV tag looks almost the same as one in start.html.template. Can you factor it out as sub template file?

The only difference seems to be whether to use `{{seek_url}}` or `query?role=seek`, but I guess you can use `{{seek_url}}` here too?